services/wallet/datastore_test.go

```diff
@@ -58,7 +58,7 @@ func (suite *WalletPostgresTestSuite) TearDownTest() {
 }
 
 func (suite *WalletPostgresTestSuite) CleanDB() {
-	tables := []string{"claim_creds", "claims", "wallets", "issuers", "promotions"}
+	tables := []string{"claim_creds", "claims", "wallets", "issuers", "promotions", "verified_wallet_outbox"}
```

Comment:
Now that the test packages have been changed there was some interference so need to add this to cleanup the table.